com.unity.perception/Editor/Randomization/Editors/RunInUnitySimulationWindow.cs

```diff
@@ -186,24 +188,30 @@ void CreateLinuxBuildAndZip()
         List<AppParam> GenerateAppParamIds(CancellationToken token)
         {
             var appParamIds = new List<AppParam>();
+            var scenario = (ScenarioBase)m_ScenarioField.value;
+            var configuration = JObject.Parse(scenario.SerializeToJson());
+            var constants = configuration["constants"];
```

Comment:
What is the use of this constants variable defined here?

Reply:
The constants variable here is a reference to the constant's section of the scenario json configuration. The loop below assigns the appropriate instance index to each created app-param by leveraging this reference.

Reply:
This constants section is consistent with the naming of the constants field defined for every scenario that inherits from the Scenario<T> abstract class.